fuzz/src/main/java/tech/pegasys/teku/fuzz/FuzzUtil.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.fuzz;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Optional;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.bls.BLSConstants;
+import tech.pegasys.teku.core.BlockProcessorUtil;
+import tech.pegasys.teku.core.StateTransition;
+import tech.pegasys.teku.core.StateTransitionException;
+import tech.pegasys.teku.core.exceptions.BlockProcessingException;
+import tech.pegasys.teku.datastructures.state.BeaconState;
+import tech.pegasys.teku.datastructures.util.BeaconStateUtil;
+import tech.pegasys.teku.datastructures.util.CommitteeUtil;
+import tech.pegasys.teku.datastructures.util.SimpleOffsetSerializer;
+import tech.pegasys.teku.fuzz.input.AttestationFuzzInput;
+import tech.pegasys.teku.fuzz.input.AttesterSlashingFuzzInput;
+import tech.pegasys.teku.fuzz.input.BlockFuzzInput;
+import tech.pegasys.teku.fuzz.input.BlockHeaderFuzzInput;
+import tech.pegasys.teku.fuzz.input.DepositFuzzInput;
+import tech.pegasys.teku.fuzz.input.ProposerSlashingFuzzInput;
+import tech.pegasys.teku.fuzz.input.VoluntaryExitFuzzInput;
+import tech.pegasys.teku.ssz.SSZTypes.SSZList;
+import tech.pegasys.teku.ssz.sos.ReflectionInformation;
+import tech.pegasys.teku.util.config.Constants;
+
+// TODO a Java FuzzHarness interface? - that way type safety can be checked at compile time
```

Comment:
Would you mind clearing out some of these TODO comments?  We should either implement the TODO's now or cut them.

Reply:
Done! Sorry to leave them sitting around.